### ISSUE TITLE
Next release

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6872]
Discussed on IRC yesterday:
```
23:03:06Lineflyer: Another topic: Shall we do a quick step? Right now master and release are close to each other but still some new things in master. How about merge to release and push another beta next week. You know...I prefer smaller release cycles
23:22:00pstorch: Good idea. Before we merge the big changes in master.


Any open PRs which should be merged before mergin to `release`?

**Comment by pstorch** [comment]
In my opinion PRs #6874 and #6862 are candidates for inclusion into the next release

**Comment by pstorch** [comment]
And #6876 ;) I'm not sure here if we should merge it first to `release` as it is a classic bug fix. But when we want to make a new feature release shortly than it might not be worth it to merge it into both branches.

**Comment by pstorch** [comment]
Ok, two of the mentioned PRs are already merged.

I would like to add PR #6882 and if my time permits I would like to work on issues #6861 to be able to fix #6697, too.

**Comment by Lineflyer** [comment]
Sounds good. 
Then I would say you should continue doing this on `master` and afterwards merge to `release`.
We should only IMHO avoid implementing huge changes on `master` (which might need finetuning efforts afterwards) before having branched to `release`.

**Comment by Lineflyer** [comment]
Due to #6884 we should do a release from ˋreleaseˋ branch asap.
Sadly I am on vacation and only online via smartphone, thus I cannot upload the release to Google Play.

Is someone else available to do it today?
I tested the searches with current master(nightly) and it looks good.

What needs to be done for release:
- Add changelog and version code in changelog_release.xml

Then the checklist from our Wiki can be used to track all needed action for release.

Would be great if someone can do this today to help our users having a good geocaching weekend.

**Comment by Lineflyer** [comment]
I raised a notification for versionCode 20171206 using message ID "status_geocaching_change" meanwhile

**Comment by kumy** [comment]
I'll do.

**Comment by kumy** [comment]
* Changelog updated

**Comment by kumy** [comment]
* Tag and Github release done.

**Comment by kumy** [comment]
* Build job launched (Let's see if add ram help in this step ;))

**Comment by kumy** [comment]
* Build success (in only 4 minutes! Thanks @mucek4)

**Comment by kumy** [comment]
- [ ] Check/Import incoming Translations from Crowdin
- [ ] Check/Insert correct version name as headline in ```changelog-release.xml```
- [ ] Build release on CI server
- [ ] Perform smoke test with resulting ```cgeo-release.apk``` before continuing
- [ ] Upload and publish ```cgeo-release.apk``` to Google Play
- [ ] Tag the release on Github
- [ ] Publish the release on Github including upload of ```cgeo-release.apk``` to Github
- [ ] Publish release on F-Droid 
- [ ] Rename/Create milestones on Github
- [ ] Trigger notification to status.cgeo.org via CI server once the release is available on Google Play
- [ ] Post release info to Facebook 
- [ ] Post release info to G+ 
- [ ] Post release info to Twitter (via @SammysHP)

**Comment by kumy** [comment]
I think it's ok, published on Google Play at 25% right now

**Comment by kumy** [comment]
I not plan to publish anything on FB/G+/Twitter, can someone else take that part please? Maybe @SammysHP ?

**Comment by kumy** [comment]
@Lineflyer I've jumped over [Step 5](https://github.com/cgeo/cgeo/wiki/Prepare-a-release#update-translations-change-log-and-version), may you please check that when you have time.

Remember, deployment set to 25%, feel free to raise as you wish, or ask me to change.
I'll be around for some hours...

**Comment by SammysHP** [comment]
I don't have access to Facebook and G+, but I can publish on Twitter once we are at 100%.

**Comment by kumy** [comment]
No crash on the console, let's publish at 100%.

**Comment by kumy** [comment]
Notification cleared.

**Comment by Lineflyer** [comment]
FB posted.

Thanks @kumy for releasing.

**Comment by Lineflyer** [comment]
Seems we have overseen another problem #6886

**Comment by Lineflyer** [comment]
I just triggered a new NB to test the fix for #6886.
I will report back later or tomorrow if its working normal for me.

**Comment by Lineflyer** [comment]
NB is working normal for me. Search still working, waypoint parsing back to normal.
@kumy Can you release another version today?

**Comment by janykey** [comment]
Hey there,

I'm just wondering if commit fc3ad60 / pr #6874 will make it into this next release.

Thanks,
JaHi

**Comment by SammysHP** [comment]
No. You can see what will be included in the release here: https://github.com/cgeo/cgeo/commits/release
There won't be any feature release this year. Maybe end of January.

**Comment by kumy** [comment]
I can in ~4hours...

On December 17, 2017 12:22:48 PM GMT+01:00, Lars <notifications@github.com> wrote:
>NB is working normal for me. Search still working, waypoint parsing
>back to normal.
>@kumy Can you release another version today?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/cgeo/cgeo/issues/6872#issuecomment-352248696

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by kumy** [comment]
Now I'm available, I start the release procedure.

- [ ] Check/Import incoming Translations from Crowdin
- [ ] Check/Insert correct version name as headline in ```changelog-release.xml```
- [ ] Build release on CI server
- [ ] Perform smoke test with resulting ```cgeo-release.apk``` before continuing
- [ ] Upload and publish ```cgeo-release.apk``` to Google Play
- [ ] Tag the release on Github
- [ ] Publish the release on Github including upload of ```cgeo-release.apk``` to Github
- [ ] Publish release on F-Droid 
- [ ] Rename/Create milestones on Github
- [ ] Trigger notification to status.cgeo.org via CI server once the release is available on Google Play
- [ ] Post release info to Facebook 
- [ ] Post release info to G+ 
- [ ] Post release info to Twitter (via @SammysHP)

**Comment by kumy** [comment]
New version released 100%. @SammysHP may you please publish on Twitter.

**Comment by SammysHP** [comment]
Done.

**Comment by Lineflyer** [comment]
I think we are done here (twice). I did not post anything related to 2017.12.17 on FB /G+ as this was already done for 2017.12.16 and the newer version mainly replaced this version.

Thanks for your efforts!!
Good to see, that release process is also working if I am unavailable...I don't want to be a bottleneck :)
